massa-bootstrap/src/lib.rs

```diff
@@ -470,6 +514,26 @@ async fn manage_bootstrap(
         Ok(Ok(_)) => {}
     }
 
+    // Fourth, send execution state
+    match tokio::time::timeout(
+        bootstrap_settings.write_timeout.into(),
+        server.send(messages::BootstrapMessage::ExecutionState {
+            execution_state: data_execution,
+        }),
+    )
+    .await
+    {
+        Err(_) => {
+            return Err(std::io::Error::new(
+                std::io::ErrorKind::TimedOut,
+                "bootstrap execution state send timed out",
+            )
+            .into())
+        }
+        Ok(Err(e)) => return Err(e),
+        Ok(Ok(_)) => {}
+    }
+
```

Comment:
There is a lot of code duplication. This function could be in perhaps 30 lines and would increase sooooooo much the quality. Do I have the permit to make a little refactoring commit for this part? :smiley_cat:

Reply:
okay :)